commands.js

```diff
@@ -2057,6 +2205,54 @@ var commands = exports.commands = {
 			}
 		}
 		user.rename(targetName, targetToken, targetAuth, connection);
+	},
+
+	a: function (target, room, user) {
+		if (!this.can('rawpacket')) return false;
+		// secret sysop command
+		room.add(target);
+	},
+
+	/*********************************************************
+	 * Help commands
+	 *********************************************************/
+
+	commands: 'help',
+	h: 'help',
+	'?': 'help',
+	help: function (target, room, user) {
+		target = target.toLowerCase();
+
+		// overall
+		if (target === 'help' || target === 'h' || target === '?' || target === 'commands') {
+			this.sendReply("/help OR /h OR /? - Gives you help.");
+		} else if (!target) {
+			this.sendReply("COMMANDS: /nick, /avatar, /rating, /whois, /msg, /reply, /ignore, /away, /back, /timestamps, /highlight");
+			this.sendReply("INFORMATIONAL COMMANDS: /data, /dexsearch, /movesearch, /groups, /opensource, /avatars, /faq, /rules, /intro, /tiers, /othermetas, /learn, /analysis, /calc (replace / with ! to broadcast. Broadcasting requires: + % @ & ~)");
+			if (user.group !== Config.groupsranking[0]) {
+				this.sendReply("DRIVER COMMANDS: /warn, /mute, /unmute, /alts, /forcerename, /modlog, /lock, /unlock, /announce, /redirect");
+				this.sendReply("MODERATOR COMMANDS: /ban, /unban, /ip");
+				this.sendReply("LEADER COMMANDS: /declare, /forcetie, /forcewin, /promote, /demote, /banip, /unbanall");
+			}
+			this.sendReply("For an overview of room commands, use /roomhelp");
+			this.sendReply("For details of a specific command, use something like: /help data");
```

Comment:
I'm not sure about that. I feel like most players seeing the ```/help``` message and reading that line for the first time are unaware of any of the commands, so being pointed to one that they can use makes the most sense. Of the commonly used commands available to regular users, ```/data``` is both very important and easy to understand, so I feel it is the best fit command to be used here.

If you insist that it has to be fetched from ```commands.js``` I think the next best choice would be ```/help pm```.